<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshViewOptionsWidget</class>
 <widget class="QDockWidget" name="MeshViewOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>532</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>215</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mesh View Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>172</width>
         <height>160</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>160</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="showAxisCheckbox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Show Axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showBBoxCheckbox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Show BBox</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showFacesCheckbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>164</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Show Mesh Faces</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showEdgesCheckbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Show Mesh Edges</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showCutsCheckbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Show Cuts</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showSurfacesCheckbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Surfaces Only</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="colorByQualityCheckbox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Color by Quality</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="clippingSettingsWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Clipping:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="clippingCheckbox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2_2">
         <property name="text">
          <string>Sync:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="syncClippingCheckbox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="xAxisWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="xCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>x-axis</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="xHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSlider" name="xSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="yAxisWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="yCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>y-axis</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="yHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSlider" name="ySlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="zAxisWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="zCheckbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>z-axis</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="zHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSlider" name="zSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label2_3">
      <property name="text">
       <string>Material Visibility Locks</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="materialsView">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clippingCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>xCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>yCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>zCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>xSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>ySliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>zSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncClippingCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>syncClippingCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scrollingCheckboxClicked(int)</slot>
  <slot>clippingCheckboxClicked(bool)</slot>
  <slot>xCheckboxClicked(bool)</slot>
  <slot>yCheckboxClicked(bool)</slot>
  <slot>zCheckboxClicked(bool)</slot>
  <slot>xSliderMoved(int)</slot>
  <slot>ySliderMoved(int)</slot>
  <slot>zSliderMoved(int)</slot>
  <slot>clippingSliderPressed()</slot>
  <slot>clippingSliderReleased()</slot>
  <slot>syncClippingCheckboxClicked(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
